pkg/operator/ceph/object/controller_test.go

```diff
@@ -348,7 +348,7 @@ var mockMultisiteAdminOpsCtxFunc = func(objContext *Context, spec *cephv1.Object
 	return &AdminOpsContext{
 		Context:               *context,
 		AdminOpsUserAccessKey: "EOE7FYCNOBZJ5VFV909G",
-		AdminOpsUserSecretKey: "qmIqpWm8HxCzmynCrD6U6vKWi4hnDBndOnmxXNsV",
+		AdminOpsUserSecretKey: "qmIqpWm8HxCzmynCrD6U6vKWi4hnDBndOnmxXNsV", // notsecret
```

Comment:
// not production secret

Reply:
The specific instructions I received in the email from gitleaks were that this must be `// notsecret`. It's a directive to a program, not a note to a person.